ansible_collections/arista/avd/molecule/eos_designs_negative_unit_tests/inventory/host_vars/duplicate-interfaces-point-to-point-services-4.yml

```diff
@@ -41,13 +45,15 @@ tenants:
           - number: 1000
         endpoints:
           - id: 26
-            nodes: [ "duplicate-interfaces-point-to-point-services-4" ]
-            interfaces: [ "Ethernet6" ]   # interface name would be Ethernet6.1000 (duplicate)
+            nodes: ["duplicate-interfaces-point-to-point-services-4"]
+            interfaces: ["Ethernet6"] # interface name would be Ethernet6.1000 (duplicate)
           - id: 25
-            nodes: [ "duplicate-interfaces-point-to-point-services-3" ]
-            interfaces: [ "Ethernet6" ]
+            nodes: ["duplicate-interfaces-point-to-point-services-3"]
+            interfaces: ["Ethernet6"]
 
 expected_error_message: >-
-  Found duplicate objects with conflicting data while generating configuration for Ethernet Interfaces defined under point_to_point_services.
-  {'name': 'Ethernet6.1000', 'encapsulation_vlan': {'client': {'encapsulation': 'dot1q', 'vlan': 1000}, 'network': {'encapsulation': 'client'}},
-  'peer_type': 'point_to_point_service'} conflicts with {'name': 'Ethernet6.1000', 'encapsulation_vlan': None, 'peer_type': 'l3_interface'}.
+  Found duplicate objects with conflicting data while generating configuration for EthernetInterfaces.
+  {'name': 'Ethernet6.1000', 'shutdown': False, 'encapsulation_vlan': {'client': {'encapsulation': 'dot1q', 'vlan': 1000},
```

Comment:
Even though this diff looks like we are catching the duplicate on different items, we are not. This is just because we don't limit the dict with `context_keys` so we see it all (and don't force the unused `encapsulation_vlan`).